### DISCUSSION TITLE
Daisyui colors (and themes)

### DISCUSSION THREAD

**Discussion by sillypnk** [saadeghi/daisyui#802] [Q&A] [answered]
Hello! i just want to know if where is daisyui's colors and themes like the color of the button i want to know what color is it of each theme (#hex)

sorry bad english, not my first language :)

**Comment by saadeghi** [comment, marked as answer]
Here you can see the preview of colors used on each theme: https://daisyui.com/docs/themes/

Here's the source (hex colors): https://github.com/saadeghi/daisyui/blob/master/src/colors/themes.js

Which generates the HSL values like this: https://unpkg.com/browse/daisyui@2.15.0/dist/themes.css

**Reply by sillypnk** [reply]
Thank you! is there a color shades for this btw? or i need to generate my own?

**Reply by saadeghi** [reply]
I'm not sure what you mean by "shade".  
If you need a complete range of colors like `blue-100` to `blue-900` those colors are still available from Tailwind CSS.  
daisyUI just adds additional color names that can be themeable.

Read this page please: https://daisyui.com/docs/colors/  
And let me know if you have any questions.

**Reply by sillypnk** [reply]
> I'm not sure what you mean by "shade". If you need a complete range of colors like `blue-100` to `blue-900` those colors are still available from Tailwind CSS. daisyUI just adds additional color names that can be themeable.
> 
> Read this page please: https://daisyui.com/docs/colors/ And let me know if you have any questions.

Like this?
![image](https://user-images.githubusercontent.com/102006493/169192434-0cada140-da94-417b-9af2-8739ccfbd0f1.png)
